### PR TITLE
fixed errors in log, activity, nodes, deploy

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
A few bugs have been fixed in the analytics API calls

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#931.0 ✘  
* 
  * build ✘  
  * test ✘

File: src/analytics/platforms/platforms.v6.service.js [hunk 1]
```diff
@@ -15,26 +15,35 @@ angular.module("ovh-api-services").service("OvhApiAnalyticsPlatformsV6", functio
     var adpResource = $resource("/analytics/platforms/:serviceName", {
         serviceName: "@serviceName"
     }, {
-        query: { method: "GET", isArray: true, cache: queryCache },
+        query: {
+            url: "/analytics/platforms",
```
Comment (antleblanc) [inline src/analytics/platforms/platforms.v6.service.js]: Specify the `url` isn't needed in the `query` action.

Comment (ganeshkumar1989) [reply src/analytics/platforms/platforms.v6.service.js]: This is removed. Thanks :)

Comment (antleblanc) [reply src/analytics/platforms/platforms.v6.service.js]: In addition your commit message should be:

```sh
fix(analytics.platforms): use the right url across actions # for instance
```

**Pull request merged**